treeherder/model/management/commands/cycle_data.py

```diff
@@ -10,6 +10,8 @@
                                      Repository)
 from treeherder.perf.models import PerformanceDatum
 
+env = environ.Env()
```

Comment:
django-environ is unused in this file now

Reply:
Fixed and rebased